psydac/linalg/stencil.py

```diff
@@ -1381,6 +1381,10 @@ def diagonal(self, *, inverse = False, out = None):
         inverse : bool
             If True, get the inverse of the diagonal. (Default: False).
```

Comment:
How about adding the same comment as in the next parameter?
```suggestion
        inverse : bool
            If True, get the inverse of the diagonal. (Default: False).
            Can be combined with sqrt to get the inverse square root.
```